Fix/display of current network switch updates

georgewrmarshall opened the pull request:
## Explanation

Some icon and color updates to ensure consistency and theme compatibility for the confirmation-network-switch and permsions-redirect components

- Also catching some other unused colors and updating

## Screenshots and Screencasts


https://user-images.githubusercontent.com/8112138/160875402-a69eb87e-af30-4dbd-9b49-9aa89e9926d2.mov


https://user-images.githubusercontent.com/8112138/160875408-db89f779-a65a-4221-a864-ada9294edb6f.mov

![Screen Shot 2022-03-30 at 8 54 02 AM](https://user-images.githubusercontent.com/8112138/160878484-5440b06d-8d94-4ae0-93dd-294cdb378838.png)
![Screen Shot 2022-03-30 at 8 55 21 AM](https://user-images.githubusercontent.com/8112138/160878489-1bfa24be-5e25-47e6-8f24-22f63596239e.png)
![Screen Shot 2022-03-30 at 8 54 11 AM](https://user-images.githubusercontent.com/8112138/160878558-0d42da6d-679c-49da-b5e5-f612b2dba8d5.png)
![Screen Shot 2022-03-30 at 8 54 44 AM](https://user-images.githubusercontent.com/8112138/160878563-d165140e-5402-4182-8e34-2effb2fb50cc.png)


<img width="1440" alt="Screen Shot 2022-03-30 at 8 05 10 AM" src="https://user-images.githubusercontent.com/8112138/160875315-6982317a-45dc-4a17-9763-971a2c2961f0.png">
<img width="1440" alt="Screen Shot 2022-03-30 at 8 06 01 AM" src="https://user-images.githubusercontent.com/8112138/160875323-859e3eaa-4738-41cd-ab69-ec27b0c81934.png">
<img width="1440" alt="Screen Shot 2022-03-30 at 8 05 32 AM" src="https://user-images.githubusercontent.com/8112138/160875359-d8efb944-5520-4a6c-8c99-89e354930d5e.png">
<img width="1440" alt="Screen Shot 2022-03-30 at 8 05 46 AM" src="https://user-images.githubusercontent.com/8112138/160875363-eb144915-b43a-4f5a-a3c5-6594e35d4d43.png">

georgewrmarshall commented on ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.stories.js:
```diff
@@ -0,0 +1,22 @@
+
+export const DefaultStory = (args) => <ConfirmationNetworkSwitch {...args} />;
+
+DefaultStory.storyName = 'Default';
```
We should try to add stories for any new components.

georgewrmarshall commented on ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js:
```diff
@@ -93,7 +93,7 @@ export default function ConfirmationNetworkSwitch({ newNetwork }) {
+            color={COLORS.TEXT_DEFAULT}
             variant={TYPOGRAPHY.H6}
             fontWeight={FONT_WEIGHT.NORMAL}
             align={TEXT_ALIGN.CENTER}
```
Use font-awesome icons in favor of svgs to ensure we have the correct licensing and allows us to access the css to apply the theme compatible colors via css

georgewrmarshall commented on ui/pages/confirmation/confirmation.js:
```diff
@@ -210,7 +210,7 @@ export default function ConfirmationPage() {
+            borderColor={COLORS.BORDER_DEFAULT}
           />
         </Box>
         <MetaMaskTemplateRenderer sections={templatedValues.content} />
```
Udpating color that no longer exists.

georgewrmarshall commented on ui/pages/confirmation/templates/switch-ethereum-chain.js:
```diff
@@ -47,7 +47,7 @@ function getValues(pendingApproval, t, actions) {
+          color: COLORS.TEXT_ALTERNATIVE,
           align: 'center',
           boxProps: {
             padding: [0, 4, 0, 4],
```
Updating color that no longer exists to theme compatible color

georgewrmarshall commented on ui/pages/permissions-connect/redirect/permissions-redirect.component.js:
```diff
@@ -21,9 +21,8 @@ export default function PermissionsRedirect({ subjectMetadata }) {
+            <i className="fa fa-check-circle fa-2x permissions-redirect__check" />
           </div>
           <SiteIcon
             icon="/images/logo/metamask-fox.svg"
```
Making the permissions redirect consistent with the confirmation network switch component as they are very similar.

brad-decker commented:
Looks good to me, we'll just wait for the base to merge to develop then rebase and merge this one.

darkwing commented:
We'll need to get this wrapped into the next release (v10.13.0).

georgewrmarshall committed:
Icon and color consistency updates and to ensure theme
 compatibility

georgewrmarshall committed:
Rebase updates

georgewrmarshall commented:
Hey @amerkadicE, that was my bad I didn't make it prominent enough. I've rebased from your PR and updated the description with new screenshots and screencasts. Let me know if you're ok to merge.

amerkadicE reviewed (approved):
LGTM
